<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serveur</class>
 <widget class="QWidget" name="Serveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveur</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>410</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEditInformation"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDiffuser">
      <property name="text">
       <string>Diffuser</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>821</width>
     <height>576</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Liste de messages en cours de diffusion : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetMessage"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSupMessage">
        <property name="text">
         <string>Supprimer un message</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonLancerDiffusion">
      <property name="text">
       <string>Lancer la diffusion</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Message d'alerte : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditAlerte"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCreerAlerte">
        <property name="text">
         <string>Créer une alerte</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Message d'information : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditInformation_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAjouterMessage">
        <property name="text">
         <string>Ajouter un message</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bulletin météo : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMeteo"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonModifBulletinMeteo">
      <property name="text">
       <string>Modifier le bulletin méteo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
